app/src/main/res/values/dimens.xml

```diff
@@ -377,7 +377,7 @@
   <dimen name="general_button_item_split_view_margin_top_left">20dp</dimen>
   <dimen name="general_button_item_split_view_margin_top_right">68dp</dimen>
   <dimen name="general_button_item_margin_top">20dp</dimen>
-  <dimen name="continue_submit_button_width">88dp</dimen>
+  <dimen name="continue_submit_button_width">180dp</dimen>
```

Comment:
I think we should change this behaviour.

Currently we are keeping the width of button to be fixed and for large fonts we have increased that to 180dp but this will start creating issues for smaller font settings.

Also ideally the width should be `wrap_content` and other attributes like margin, padding, fontSize should handle the UI.

For this specific case following changes to the `Continue Button` should mostly fix the issue:

1. Instead of fixed width as 180dp we should make `layout_width` as `wrap_content`.
2. We should keep a padding of `10dp` as per mocks - https://xd.adobe.com/view/3dca36c2-5115-419c-b25e-0f10526b077c-6899/screen/0fe704c1-f8c7-4abc-8b04-2a0ed1285db4/


To test this fully make sure you generate screenshots of small, medium as well as large font size.
You might need to look at this link about how to use mocks correctly: https://github.com/oppia/oppia-android/wiki/Working-on-UI

Reply:
@Ryggs when addressing comments, please reply rather than marking them as resolved (generally, only the origin comment author should resolve their comment since they need to check that you've addressed the original point, and it's often easy to miss comments that were closed by other people).

Reply:
@Ryggs Assign it back to me whenever you need a reply.

Reply:
Hello Rajat, this is the error I am getting after replacing continue_submit_button_width fully and instead using wrap_content in the layout file
![image](https://user-images.githubusercontent.com/31986629/181808211-bd107b85-d0a3-4cc3-a376-c10b1650790d.png)

THis is how I did it.
![image](https://user-images.githubusercontent.com/31986629/181808300-fb8d0135-c80f-41a8-8ab7-1a7acf26ba49.png)
Replaced  the line calling the max_width dimension and replaced with app_layout_width and used wrap_content as that is the only way to do it perhaps? 
I'll appreciate your feedback PTAL

Reply:
@Ryggs I believe attributes are case sensitive. It seems like you might've unintentionally capitalized the 'w' for width. Plus, shouldn't this be an android not an app attribute? E.g. ``android:layout_width``.

Reply:
Let me try this out now

Reply:
@Ryggs You can remove this value from all `dimens` now as it might not be getting used.

Reply:
Hi @rt4914 I have removed this value. PTAL